tests-lua/mandelbrot/Makefile

```diff
@@ -1,4 +1,4 @@
-FILE=mandelbrot.lua-6.lua
+FILE=mandelbrot.lua-3.lua
```

Comment:
When the change from 6 to 3?

Reply:
It fails even in "normal" CHERI (without the splitter), so we can skip it for now, I *think*.

Reply:
What fails and why does changing 6 to 3 change anything?

Reply:
`popen` is not supported, it is a kind of know thing in different architectures. But usually people have this issue on Macs or other distros. I have tried this even without the splitter and I can't `popen` on CHERI. We are using `lua v5.4`. Those tests have been modified to be compatible with `v5.3`. Nothing is said about `mandelbrot.lua-6.lua`, though.

Reply:
Since this does not seem to be a splitter's issue, I thought we could have skipped it for now.

Reply:
Remember that I don't know the code :) I don't know what "6" -> "3" does. Does it get rid of popen? change which version of lua this runs on? or ... ?

Reply:
"3" does not use `popen` as far as I can tell.

Reply:
Each number refers to different v[ersions/complexities](https://benchmarksgame-team.pages.debian.net/benchmarksgame/performance/mandelbrot.html)(?). In this case, should be "children".

Reply:
Ah I see!